### PR TITLE
fix sync of global example values

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
This will now generate the combined values.yaml for the kubermatic-installer repository based on the values found in each chart

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: api/hack/sync-charts.sh [hunk 1]
```diff
@@ -26,16 +27,39 @@ fi
...
+touch ${TARGET_DIR}/values.yaml
+
 for CHART in ${CHARTS}; do
   echo "syncing ${CHART}..."
   # doing clean copy
   rm -rf ${TARGET_DIR}/charts/${CHART}
   cp -r ${CHARTS_DIR}/${CHART} ${TARGET_DIR}/charts/${CHART}
+
+  echo "# ====== ${CHART} ======" >> ${TARGET_DIR}/values.yaml
+  cat "${CHARTS_DIR}/${CHART}/values.yaml" >> ${TARGET_DIR}/values.yaml
```
Comment (alvaroaleman) [inline api/hack/sync-charts.sh:45]: Shouldn't this be a single `>` insead of `>>`? therwise we will end up with a pretty long `values.yaml`

Comment (alvaroaleman) [reply api/hack/sync-charts.sh:45]: ah nvm, missed the `rm` above

Comment (mrIncompetent) [reply api/hack/sync-charts.sh:45]: We combine all chart specific values into a global values.yaml. So `>>` is correct here

**Comment by mrIncompetent** [comment]
@alvaroaleman PTAL

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**